bin/wp-now

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+
+if [[ $(uname -s) == 'Darwin' ]]; then
+  SCRIPT_PATH=$(realpath "$0")
+else
+  SCRIPT_PATH=$(readlink -f "$0")
+fi
+SCRIPT_DIR=$(dirname "$SCRIPT_PATH")
+
+exec /usr/bin/env node --loader $SCRIPT_DIR/../packages/nx-extensions/src/executors/built-script/loader.mjs $SCRIPT_DIR/../dist/packages/wp-now/main.js "$@"
```

Comment:
It might actually be preferable to run `nx preview` in this context:

```diff
-exec /usr/bin/env node --loader $SCRIPT_DIR/../packages/nx-extensions/src/executors/built-script/loader.mjs $SCRIPT_DIR/../dist/packages/wp-now/main.js "$@"
+cd $SCRIPT_DIR/..
+npx nx preview wp-now "$@"
```

However, doing so produces another crash:

<img width="1205" alt="image" src="https://github.com/WordPress/wordpress-playground/assets/36432/4741089d-9850-46ff-a501-4379ed937fb6">

We'd need to sort out the node version switching.

Reply:
that one can likely be solved with `nvm use`

Reply:
@adamziel a68e4fc6 seems to do the trick!

Reply:
This needs to account for devs not using nvm, but other than that it looks good.